test/test_ops.py

```diff
@@ -2195,7 +2195,16 @@ def test_fake_crossref_backward_no_amp(self, device, dtype, op):
     @skipOps('TestFakeTensor', 'test_fake_crossref_backward_amp', fake_backward_xfails | fake_autocast_backward_xfails)
     def test_fake_crossref_backward_amp(self, device, dtype, op):
         self._test_fake_crossref_helper(device, dtype, op, torch.cuda.amp.autocast)
-
+        
+    @ops([op for op in op_db if op.is_factory_method])
+    def test_strided_layout(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+        for sample in samples:
+            kwargs = sample.kwargs.copy()
+            kwargs['layout'] = torch.strided  
```

Comment:
With or without this assignment, the test will pass. all factory functions being tested are with the default layout=`torch.strided`.

if i understand correctly, the scope of this test is to test if the result tensor has the layout being specified.
The specified layout can be `torch.strided` or something else.  Doesn't we need to cover the second case to be safe?

Reply:
See the PR linked in the original issue. The test could be more general, but that's out of the scope of this PR

Reply:
@lezcano @akdong91 Something like this? 
@ops([op for op in op_db if op.is_factory_function])  
def test_tensor_layouts(self, device, dtype, op):
    layouts = [torch.strided, torch.sparse_coo] 
    samples = op.sample_inputs(device, dtype)

    for sample in samples:
        for layout in layouts:
            kwargs = sample.kwargs.copy()
            kwargs['layout'] = layout
            result = op(sample.input, *sample.args, **kwargs)
            self.assertEqual(result.layout, layout)

Reply:
Not for this PR.

Reply:
> See the PR linked in the original issue. The test could be more general, but that's out of the scope of this PR

That is fair enough.

Reply:
@akdong91  @lezcano  i beleiev that the following commit Only tests torch.strided layout. Modifies kwargs copy rather than original and Checks result layout matches strided layout kwarg.